### PR TITLE
Schedule for client.py

### PR DESCRIPTION
**Pull request by nirupaman**
@schedule

### CODE REVIEW

**Comment by nirupaman** [comment]
@farhan7500 please review ..Whether i have to create test for sdk ?

**Comment by nirupaman** [comment]
@please review

**Comment by nirupaman** [comment]
@farhan7500

File: hpe3par_sdk/client.py [hunk 1]
```diff
@@ -3482,195 +3724,55 @@ def toggleRemoteCopyConfigMirror(self, target, mirror_config=True):
...
+        :param schedule_name - The name of the schedule to delete
+        :type - string
+        """
+        return self.client.deleteSchedule(schedule_name)
 
-    def flashCacheExists(self):
+    def scheduleExists(self, name):
         try:
-            self.getFlashCache()
+            self.getSchedule(name)
```
Comment (farhan7500) [inline hpe3par_sdk/client.py]: What are we checking here?

Comment (nirupaman) [reply hpe3par_sdk/client.py]: have push the latest changes

**Comment by farhan7500** [comment]
Please remove all changes unrelated to your commit, only then can I review

**Comment by nirupaman** [comment]
Run for create, get, schedule exist and delete schedule

root@cssosbe03-b12:/home/nirun/niru_ansiblecopy/hpe3par_python_sdk# vim test_snap.py
root@cssosbe03-b12:/home/nirun/niru_ansiblecopy/hpe3par_python_sdk# python test_snap.py
/usr/local/lib/python2.7/dist-packages/urllib3/connectionpool.py:857: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
/usr/local/lib/python2.7/dist-packages/urllib3/connectionpool.py:857: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
/usr/local/lib/python2.7/dist-packages/urllib3/connectionpool.py:857: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)

File: hpe3par_sdk/client.py [hunk 2]
```diff
@@ -2598,7 +2805,37 @@ def getRemoteCopyGroup(self, name):
...
+        :returns: list of volumes in a Remote Copy Groups
+
+        """
+        remoteCopyGroupVolumes = []
+        remoteCopyGroupVolumes_list = self.client.getRemoteCopyGroupVolumes(remoteCopyGroupName)['members']
+        for remoteCopyGroupVolume in remoteCopyGroupVolumes_list:
+            remoteCopyGroupVolumes.append(Volumes(remoteCopyGroupVolume))
+        return remoteCopyGroupVolumes
+
+    def getRemoteCopyGroupVolume(self, remoteCopyGroupName, volumeName):
```
Comment (farhan7500) [inline hpe3par_sdk/client.py:2826]: Why do we have remote copy changes in this PR?

**Review by farhan7500 — CHANGES_REQUESTED** [review]

**Comment by nirupaman** [comment]
Repo deleted.

**Pull request closed without merging**